dynamic/api_test.py

```diff
@@ -35,7 +37,8 @@ def fetch_input_url(cls):
         # Ask the user for endpoint if not present in request_url
         if not (has_endpoint):
             if request_url[-1] == "/":
-                endpoint = input("Input endpoint " + "(Without the starting slash): ")
+                endpoint = input("Input endpoint " +
+                                 "(Without the starting slash): ")
             else:
                 endpoint = input("Input endpoint (With the starting slash): ")
             request_url += endpoint
```

Comment:
I'd recommend using this,  we can allow slash in the beginning. we just need to remove it after taking input.
```py
if not has_endpoint:
  endpoint = input("Input endpoint ")
  endpoint = endpoint[1:] if endpoint.startswith("/") else endpoint
  request_url += endpoint
```